packages/ember-views/tests/views/view/remove_test.js

```diff
@@ -10,44 +9,8 @@ import viewKeyword from 'ember-htmlbars/keywords/view';
 import { objectAt } from 'ember-runtime/mixins/array';
 
 
-// .......................................................
-// removeChild()
```

Comment:
I was uncertain if removeAllChildren is a "normal" view method or just for container view.